libvmaf/src/x86/cpuid.asm

```diff
@@ -44,11 +44,10 @@ cglobal cpu_cpuid, 0, 5, 0, regs, leaf, subleaf
 %endif
     RET
 
-cglobal cpu_xgetbv, 0, 0, 0, xcr
+cglobal cpu_xgetbv, 0, 4, 0, regs, xcr
+    mov        r3, regsmp
```

Comment:
Isn't r3 the same as rcx on unix-64-bit abi?

https://github.com/discord/vmaf/blob/discord/libvmaf/src/ext/x86/x86inc.asm#L549

I think using r4 (like cpu_cpuid does) never collides with rax, rcx, or rdx

Reply:
I was wondering why `cpu_cpuid` skipped it, thanks for pointing this out.